moveit_msgs/RobotTrajectory trajectory
---
bool done
